<launch>
	<rosparam command="load" file="$(find panthera_locomotion)/params/steer_params.yaml"/>

	<node name="lf_steering_motor" type="lf_steer.py" pkg="panthera_locomotion" output="screen"/>
</launch>
